HACKING.txt

```diff
@@ -161,15 +145,15 @@ Coding Style
 Running Tests
 --------------
 
-- To run all tests for Pyramid on a single Python version, run ``nosetests``
-  from your development virtualenv (See *Using a Development Checkout* above).
+- To run all tests for Pyramid on a single Python version from your development
+  virtual environment (See *Using a Development Checkout* above), and assuming
+  you have installed the test requirements::
+
+   $ $VENV/bin/py.test --strict pyramid/
```

Comment:
Recommending to use tox instead is fine. That's how all of the core devs run the tests (at least @bertjwregeer and I) both use commands like `tox -e py27`, `tox -e py2-cover,py3-cover,coverage`, etc. The tox.ini uses (and will continue to use) nose and coverage.